app/services/geo_discovery/document_builder/basic_metadata_builder.rb

```diff
@@ -80,7 +80,10 @@ def subject
         # Use the standard resource decorator to retreive title.
         # @return [String] title
         def title
-          resource_decorator.model.decorate.title.try(:first)
+          # This convoluted set ot method calls is required because, in this case, we need the
+          # actual model decorator rather than the geoblacklight metadata decorator.
+          decorated_title = resource_decorator.model.decorate.title.try(:first)
```

Comment:
That's odd, from what I would assume, this would just be delegated to the underlying resource decorator:
https://github.com/pulibrary/figgy/blob/master/app/services/geo_discovery/document_builder.rb#L31

Reply:
Perhaps this is just related to my misunderstanding of https://github.com/pulibrary/figgy/blob/master/app/decorators/geoblacklight_metadata_decorator.rb#L20